README.md

```diff
@@ -195,6 +195,30 @@ const {storedValue, setValue} = useLocalStorage('ID', 'abc123');
 }
 ```
 
+---
+
+### useToggle
+
+**Description** Simple hook that returns a boolean value and a function that toggles that value to the inverse of its previous state.
```

Comment:
nit, maybe? Wouldn't `toggle` toggle the value to the inverse of the current state? This is probably a stupid english thing

Reply:
Haha, now that I read your comment I see what you mean.

Reply:
Yep, good catch